CHANGELOG

```diff
@@ -1,3 +1,6 @@
 1.0.0-b6
 
+* Use SFINAE on return values
+* Use beast::error_code instead of nested types
```

Comment:
Edits to text files like TODO.txt and CHANGELOG are not necessarily temporally consistent with edits made to source code :-) 

For CHANGELOG what matters is the accuracy at the time of the commit that sets the version.